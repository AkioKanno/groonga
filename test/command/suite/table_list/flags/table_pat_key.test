table_create bookmarks TABLE_PAT_KEY
table_list
