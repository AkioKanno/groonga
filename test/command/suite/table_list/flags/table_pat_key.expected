table_create bookmarks TABLE_PAT_KEY
[[0,0.0,0.0],true]
table_list
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        "id",
        "UInt32"
      ],
      [
        "name",
        "ShortText"
      ],
      [
        "path",
        "ShortText"
      ],
      [
        "flags",
        "ShortText"
      ],
      [
        "domain",
        "ShortText"
      ],
      [
        "range",
        "ShortText"
      ]
    ],
    [
      256,
      "bookmarks",
      "db/db.0000100",
      "TABLE_PAT_KEY|PERSISTENT",
      "null",
      "null"
    ]
  ]
]
